code/modules/mapping/reader.dm

```diff
@@ -401,6 +401,8 @@ var/global/dmm_suite/preloader/_preloader = new
 		var/value = attributes[attribute]
 		if(islist(value))
 			value = deepCopyList(value)
+		if(istext(value))
+			value = eval("\"[value]\"")
```

Comment:
it's the least chinsky way to do that, and its not like we don't use `static` or `set waitfor = 0`